tests/readme.md

```diff
@@ -152,3 +152,21 @@ value `amqp`:
 ```sh
 mvn verify -Prun-tests -Dhono.messaging-infra.type=amqp
 ```
+
+### Run Tests for a particular Protocol Adapter only
+
+By default, all protocol adapter listed in the overview above are started as part of running the integration tests.
+However, when working on a particular protocol adapter, only the integration tests verifying that adapter's behavior
+will be of interest and thus starting all other adapters seems unnecessary.
+
+The following Maven profiles can be used to run the integration tests with a single protocol adapter only:
+
+* Profile: `amqp-only`, Maven property: `hono.amqp-adapter.disabled`
```

Comment:
That seems misleading here: `amqp-only` will only run the AMQP adapter tests whereas setting `hono.amqp-adapter.disabled` to `true` will skip them.